lib/src/rules/directives_ordering.dart

```diff
@@ -271,7 +271,14 @@ class _Visitor extends SimpleAstVisitor<void> {
     _checkSectionInOrder(lintedNodes, relativeExports);
 
     var projectName = project?.name;
-    if (projectName != null) {
+    if (projectName == null) {
+      // Not a pub package. Package directives should be sorted in one block.
```

Comment:
It would be good to have some tests for this case, but I don't know whether our current test framework supports test cases in non-pub packages.

Reply:
Thanks.

I hadn't noticed the test for this rule under `integration`; now I did notice it and tried to add coverage--but didn't figure out a way to trigger this codepath.